src/widgetastic_patternfly5/charts/boxplot_chart.py

```diff
@@ -0,0 +1,104 @@
+from widgetastic.widget import ParametrizedLocator
+from widgetastic.widget import View
+
+from .legend import Legend
+
+
+class BoxPlotChart(View):
```

Comment:
This is inspired by `LineChart`.  Most of calls are similar. 
I think, it will be great if you inherit it and overwrite respective changes only.

Reply:
it will not